test/root.test.js

```diff
@@ -45,6 +46,7 @@ test("non-empty root", () => {
     </main>
   `
 })
+*/
```

Comment:
Hmm, I think think there is some instances we want this test, no?  Say for example, `document.body` at root, and you have a `<script>` inside of it.